teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionEngineOptions.java

```diff
@@ -15,22 +15,19 @@
 
 import static tech.pegasys.teku.config.TekuConfiguration.Builder;
 
-import java.util.ArrayList;
-import java.util.List;
 import picocli.CommandLine.Option;
 
 public class ExecutionEngineOptions {
 
   @Option(
-      names = {"--Xee-endpoints", "--Xee-endpoint"},
+      names = {"--Xee-endpoint"},
       paramLabel = "<NETWORK>",
-      description = "URLs for Execution Engine nodes.",
-      split = ",",
-      arity = "0..*",
+      description = "URL for Execution Engine node.",
+      arity = "1",
       hidden = true)
-  private List<String> eeEndpoints = new ArrayList<>();
+  private String eeEndpoint = null;
```

Comment:
can we just buy some consonants? `executionEngineEndpoint`?

Reply:
Yeah seems like a good idea.